src/components/app-text-area/AppTextArea.tsx

```diff
@@ -13,6 +15,8 @@ interface AppTextAreaProps
   maxLength?: number
   errorMsg?: string
   value?: string
+  textFieldStyles?: SxProps
+  withHelperText?: boolean
```

Comment:
It is redundant in this file. Delete it.

Reply:
it passes through the props to TextField